### PR TITLE
Triggerize line vertical fix

### PR DESCRIPTION
**Pull request by jyg**
when  selecting a line between two objects and then pressing ctrl + t, the new trigger box (or pd nop~) is created in the wrong place if the first object (source) height is bigger than default object height (due to its gop properties enabled).  
image.
The proposed modification in g_editor_extras.c fixes this issue.

### CODE REVIEW

**Review by Ant1r — COMMENTED** [review]
cool!
maybe you could avoid the first "merge" commit, by restarting your branch with a `git --reset upstream/master`, then force push?

File: src/g_editor_extras.c [hunk 1]
```diff
@@ -368,24 +368,31 @@ static int triggerize_line(t_glist*x, t_triggerize_return*tr)
...
+            posSinkY /= x->gl_zoom;
             nio = obj_ninlets(obj2);
             posSink = posLeft + (posRight - posLeft - IOWIDTH) * dst_in / ((nio==1)?1.:(nio-1.));
 
+		/* get height of the box that will be inserted */
+	    boxHeight = glist_fontheight(x) / x->gl_zoom + 4; /* ATOM_BMARGIN = 4 */
+
             posx = (posSource + posSink) * 0.5;
-            posy = (obj1->te_ypix + obj2->te_ypix) >> 1;
+            posy = (posSourceY + posSinkY)* 0.5 - boxHeight/2;
```
[the remark this answers is not in the document]
Comment (jyg) [reply src/g_editor_extras.c:395]: Yes, sorry for missing spaces. Regarding branch restarting, I'm still a beginner with github, but I'll give it a try.

[the remark this answers is not in the document]
Comment (Ant1r) [reply src/g_editor_extras.c:395]: ok, maybe I could have been more explicit: 
```
cd [your_local_fork]
git checkout triggerize_line_vertical_fix
git fetch upstream/master
git reset --hard upstream/master
--- re-do your work, re-commit ---
git push --force
```

[the remark this answers is not in the document]
Comment (umlaeute) [reply src/g_editor_extras.c:395]: or use `git reset --mixed origin/master` so you don't lose your changes ;-)

[the remark this answers is not in the document]
Comment (umlaeute) [reply src/g_editor_extras.c:395]: as i don't fully understand the subtle differences in the use of `* 0.5` vs `/ 2`, shouldn't the actual code be simplifid to what the original code uses:

> posy = (posSourceY + posSinkY - boxHeight) >> 1;

**Comment by jyg** [comment]
closed pull request. substituted with https://github.com/pure-data/pure-data/pull/1243

**Pull request closed without merging**